docs/troubleshooting/repository-scripts.md

```diff
@@ -0,0 +1,71 @@
+Troubleshooting Guide for Yum Repository Scripts
+================================================
+
+About This Guide
+----------------
+
+
+The *repo1.grid.iu.edu* and *repo2.grid.iu.edu* hosts contain the OSG Yum software repositories plus related services and tools. In particular, the *mash* software is used to download RPMs from where they are built (at the University of Wisconsin–Madison), and there are some associated scripts to configure and invoke *mash* periodically. Use this guide to monitor the *mash* system for problems and to perform basic troubleshooting when such problems arise.
+
+Monitoring 
+----------
+
+To monitor the repository hosts for proper *mash* operation, do the following steps on each host:
+
+   1. Access http://repo1.grid.iu.edu/web/log/ or http://repo2.grid.iu.edu/web/log/ to view logs from *mash* updates
+   1. Examine the “Last modified” timestamp of all of the *update\_repo.\*.log* files  
+       **Note:** The timestamps are displayed in UTC, so convert to local time accordingly</p>
+   1. If the timestamps are all less than 2 hours old, life is good and you can skip the remaining steps below
+   1. Otherwise, examine the “Last modified” timestamp of the *update\_all\_repos.err* file
+   1. If the *update\_all\_repos.err* timestamp is current, there may be a *mash* process that is hung; see the Troubleshooting steps below
+   1. If **all** timestamps are more than 6 hours old, something may be wrong with cron or its mash entries:
+      1. If possible, verify that cron is running and that the cron entries for mash are still present; if not, try to restore things
```

Comment:
@brianhlin,

> edquist IIRC, you're the original author

Nope! That was TimC's word-smithing:

https://github.com/edquist/osg-twiki-backup/commit/39344190367c297851822a7cac6e4992d3e24929#diff-2a528db9d3952a5339e2a69509397978R21

But, i presume his "if not" meant something like "if cron is not running or the cron entries for mash are not still present", and then "try to restore things" meant something like, "try to make it so that cron is running again, and/or restore the missing cron entries for mash (either by un-commenting lines from the repo cron file, or by restoring the original repo cron file from the rpm)"

But yeah, the suggestion to drop "If possible, " does make the "if not" less confusing.